app/src/main/java/com/teamagam/gimelgimel/app/map/view/DynamicLayerActionFragment.java

```diff
@@ -17,16 +16,23 @@
 public class DynamicLayerActionFragment
     extends DrawGeometryActionFragment<DynamicLayerEditViewModel> {
 
+  private static final String DYNAMIC_LAYER_ID_KEY = "dynamic_layer_id_key";
```

Comment:
Second occurance, extract to `Constants`? Or at least import it in one of the classes that use it, instead of declaring it twice